deploy/deploy-gcs.yml

```diff
@@ -40,6 +40,8 @@
             - gcs-gd2-services.yml
             - gcs-csi.yml
```

Comment:
rename `gcs-csi.yml` to `gcs-fs-csi.yml`

Reply:
Done